content/about/luke/_index.md

```diff
@@ -20,7 +20,7 @@ large organizations), and that's why I initially led (along with my fearless
 fellow founding members) the push to ensure there was a single,
 vendor-neutral, and community-led fork of Mapbox GL JS, and I'm really
 excited to see the community and engagement grow over the last years. So
-far, I've been most involved in the background, from Twitter-spamming at the
+far, I've been most involved in the background, from X-spamming at the
```

Comment:
I would prefer this remains "Twitter"—it was Twitter at the time. 😄

Reply:
done